study-builder/studies/study-example.conf

```diff
@@ -79,10 +79,11 @@
   ],
 
   # The list of languages supported by a study, the order is unimportant.
-  # Each language is a 2-letter ISO language code, e.g. "en", "ru", "fr" etc and set isDefault to true/false
+  # Each language is a 2-letter ISO language code, e.g. "en", "ru", "fr" etc , language name e.g. "English" and isDefault (true/false)
   # Optional, can be left out or set to null.
   "supportedLanguages": [
       "language": "string",
+      "name": "string",
```

Comment:
as of now name and isDefault are optional in study.conf 
if needed we can enforce to have them entered if supportedLanguages are set !!!!!

Reply:
I think that's fine for now, study's can set their `name`s if they're using that.